etc/skydive.yml.default

```diff
@@ -63,8 +63,8 @@ analyzer:
 
   # Flow storage engine
   flow:
-    # Available: elasticsearch, orientdb, sample
-    # backend: elasticsearch
+    # Available: myelasticsearch, myorientdb
```

Comment:
please use:

Name of the backend definition, ex: myelasticsearch, mybackend, etc.

otherwise we think that is only those that are available

Reply:
done